greenwave_monitor/scripts/ncurses_dashboard

```diff
@@ -115,7 +121,11 @@ echo "Monitor process started with PID: $MONITOR_PID"
 # Launch ncurses frontend in the foreground
 echo "Starting ncurses TUI..."
 echo "Controls: a=Add Topic, r=Remove, f=Set Frequency, c=Clear Freq, q=Quit"
-python3 -m greenwave_monitor.ncurses_frontend
+FRONTEND_ARGS=()
```

Comment:
Maintaining this will become painful if we keep adding more args to the frontend ... personally I think this is OK for now and I don't want to complicate arg parsing for a single boolean ... but can you leave a note to pester ourselves to add proper frontned arg parsing if we add a second frontend arg?